### DISCUSSION TITLE
Formalizing Process for Adding Contributors

### DISCUSSION THREAD

**Discussion by thaddmt** [maplibre/maplibre-gl-js#581] [Ideas]
Currently our MapLibre [dev group](https://github.com/orgs/maplibre/teams/devs) only has 12 contributing members which can help in merging code. Does anyone have any thoughts or ideas about formalizing a process for adding new members to the group who can help review and merge open PRs?

One proposal made by @nyurik would be to have separate dev groups per project to limit the scope that these groups have access to and would likely allow us to add contributors more quickly.

**Comment by HarelM** [comment]
I've added 2 developers to this repo but not as part of any team.
I would say that if someone send at least 2 PRs  or answers to at least 4 issues they are entitled.

**Comment by wipfli** [comment]
Sounds good. Let's use "they are entitled" instead of "he is entitled".

**Reply by HarelM** [reply]
Sorry, bad translation from Hebrew... I've fixed it...
